instructions/move_the_taxi.md

```diff
@@ -21,7 +21,7 @@ The algorithm:
 
 ## Display the counter
 
-Display the value of `taxiLocationCounter` on the screen using the `displayMessage` function when the forward arrow is pressed.
+Create a global variable called `taxiLocationCounter` initialize it to 1. Increment it by 1 when the forward key is pressed, display the value of `taxiLocationCounter` on the screen using the `displayMessage` function.
```

Comment:
"Create a global variable called `taxiLocationCounter` initialize it to 1. Increment it by 1 when the forward key is pressed, display the"

->

"Create a global variable called `taxiLocationCounter` and initialize it to 1. Increment it by 1 when the forward key is pressed, and display the"